packages/fxa-auth-server/lib/senders/emails/fluent-localizer.ts

```diff
@@ -101,13 +98,13 @@ class FluentLocalizer {
 
     const isLocaleRenderedRtl = RTL_LOCALES.includes(selectedLocale);
     if (isLocaleRenderedRtl) {
-      const body = document.getElementsByTagName('body')[0];
+      const body = document.getElementsByClassName('body-80')[0];
```

Comment:
This is looking pretty specific. How fragile is it? Maybe we can leave a comment in the code

Reply:
Hmm yeah, is `byTagName` not working any longer?

Reply:
I think storybook displays everything that's inside the body tag and it's own head and body elements which is why the `rtl` styles were not getting applied to the elements. I can definitely leave a note about it

Reply:
@Yogita98 @LZoog Here's an idea: instead of trying to satisfy Storybook in fluent-localizer (should we be trying to satisfy a dev component in prod code?), and instead of needing to add a Story with an RTL locale just to see what RTL looks like, how about we extend Storybook!

I've created this quick little toolbar extension that allows you to set the text direction in any Story (it just adds the `rtl` class). It looks like this:

![Screen Shot 2021-08-05 at 10 42 30 AM](https://user-images.githubusercontent.com/6392049/128360078-ced4118d-30f8-4869-8922-5cf831ca110a.png)

It should persist across all Stories, too, so every email template can be previewed without needing to add additional RTL-locale stories.

Here's the code: https://gist.github.com/jodyheavener/18665a6109d9373d2f752d28cb5b4ff3

Let me know what you think.

Reply:
@jodyheavener I like that idea, nice thinking!

I wonder if we should have a front-page in Storybook similar to the front-page of the design guide with something really brief about the `doc` label, a note about this, and anything else relevant. Not something for this PR, though.

Reply:
Also, related, Jody mentioned this in his review comment but I want to make sure it's not missed, I'm also seeing a Fluent warning `Missing translations in en-US: Your Friendly Reminder: How To Complete Your Sync Setup` when running Storybook.

Reply:
> are you saying we should also remove the sample AR Fluent file/AR locale story?

That was part of my intent, yeah, as it's another thing we have to keep in sync if the template itself evolves. But I can see how having a non-English locale even for one story could be nice to have around. At any rate I would still hope to avoid modifying the fluent localizer to suit the storybook setup, though, which we no longer appear to be doing 👍 

Thanks for adding that Storybook toolbar extension @Yogita98!

Reply:
Sure, thanks for sharing the gist @jodyheavener